pkg/languagedetection/internal/detectors/util.go

```diff
@@ -0,0 +1,13 @@
+package detectors
+
+import (
+	"path"
+	"strconv"
+
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+)
+
+func getExePath(pid int) string {
+	hostProc := util.HostProc()
+	return path.Join(hostProc, strconv.Itoa(pid), "exe")
+}
```

Comment:
I think we can store hostProc in the go detector once and remove the need to re-evaluate it everytime

Reply:
Makes sense. I added this functionality.